generator/Makefile

```diff
@@ -367,3 +371,10 @@ $(MIBDIR)/readynas:
 $(MIBDIR)/readydataos:
 	@echo ">> Downloading readydataos"
 	@curl $(CURL_OPTS) -o $(MIBDIR)/readydataos "$(READYDATAOS_URL)"
+
+$(MIBDIR)/SCHLEIFENBAUER-PRODUCTS-MIB.txt $(MIBDIR)/SCHLEIFENBAUER-DATABUS-MIB.txt $(MIBDIR)/SCHLEIFENBAUER-SMI.txt:
+	$(eval TMP := $(shell mktemp))
+	@echo ">> Downloading schleifenbauer to $(TMP)"
+	@curl $(CURL_OPTS) -o $(TMP) $(SCHLEIFENBAUER_URL)
+	@unzip -j -d $(MIBDIR) $(TMP) SCHLEIFENBAUER-PRODUCTS-MIB.txt SCHLEIFENBAUER-DATABUS-MIB.txt SCHLEIFENBAUER-SMI.txt
```

Comment:
The SCHLEIFENBAUER-PRODUCTS-MIB.txt seems not to be included in this zipfile, only in the older https://docs.schleifenbauer.eu/Interfaces/SNMP/PDU_mib_244.zip. If the file is necessary we'd need to get it from the 2nd zipfile.